.licenseignore

```diff
@@ -0,0 +1,11 @@
+**/.swiftformat
+*.pb.swift
+*.txtpb
+.gitignore
+.gitmodules
+.swiftformat
+CITATION.cff
+Package.resolved
+Package.swift
+Snippets/*
```

Comment:
We have a non-license [header](https://github.com/apple/swift-homomorphic-encryption/blob/4008ac42d9cc29b7ade2da1517dc5ed2b6fc3a6b/Snippets/HomomorphicEncryption/BasicsSnippet.swift#L1-L3):

```
// Example showing the basics.

// snippet.hide
```